samples/SimpleDataSource/HelloWorldCustomDataProcessor.cs

```diff
@@ -0,0 +1,159 @@
+﻿using Microsoft.Performance.SDK;
+using Microsoft.Performance.SDK.Extensibility;
+using Microsoft.Performance.SDK.Processing;
+using SampleCustomDataSource.Tables;
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace SampleCustomDataSource
+{
+    //
+    // This is a sample Custom Data Processor that processes simple text files.
+    //
+    // Custom Data Processors are created in Custom Data Source and are used to actually process the files.
+    // An instance of the Processor is created for each file you open whereas only one instance of the Custom Data Source is ever created.
+    //
+    // Data processor is responsible for instantiating the proper tables based on what the user has decided to enable.
```

Comment:
1) nit: **The** data processor

2) How does the processor know which tables the user has enabled?

Reply:
A CDS can override the OnTableEnabled method to get call backs when the table is enabled. Not required for all CDS